### PR TITLE
add more example usage to varlink endpoints

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/varlinkapi/images.go [hunk 1]
```diff
@@ -508,7 +508,7 @@ func (i *LibpodAPI) PullImage(call iopodman.VarlinkCall, name string) error {
 // ImageExists returns bool as to whether the input image exists in local storage
 func (i *LibpodAPI) ImageExists(call iopodman.VarlinkCall, name string) error {
 	_, err := i.Runtime.ImageRuntime().NewFromLocal(name)
-	if errors.Cause(err) == libpod.ErrNoSuchImage {
+	if errors.Cause(err).Error() == libpod.ErrNoSuchImage.Error() {
```
Comment (mheon) [inline pkg/varlinkapi/images.go]: Woah - this didn't work without forcing a string comparison?

Comment (mheon) [reply pkg/varlinkapi/images.go]: Oh - it's image.ErrNoSuchImage

Comment (mheon) [reply pkg/varlinkapi/images.go]: (Where image is libpod/image)

Comment (baude) [reply pkg/varlinkapi/images.go]: no, i think the error being returned has been been munged sufficiently?

Comment (mheon) [reply pkg/varlinkapi/images.go]: I think you can drop the .Error() if you swap to `errors.Cause(err) == image.ErrNoSuchImage`

Comment (mheon) [reply pkg/varlinkapi/images.go]: (We should probably drop `libpod.ErrNoSuchImage` to prevent this from happening int he future)

**Comment by mheon** [comment]
LGTM

**Pull request merged**